tests/integration/backend_test.py

```diff
@@ -762,50 +765,81 @@ def test_device_state(
 
 @pytest.mark.skipif(skip_remote_tests, reason=REASON)
 @pytest.mark.parametrize(
-    "authenticated_quum_backend", [{"device_name": "H1-1SC"}], indirect=True
+    "authenticated_quum_backend_qa", [{"device_name": "H1-1SC"}], indirect=True
+)
+@pytest.mark.parametrize(
+    "language",
+    [
+        Language.QASM,
+        Language.QIR,
+    ],
+)
+@pytest.mark.timeout(120)
+def test_wasm_qa(
+    authenticated_quum_backend_qa: QuantinuumBackend, language: Language
+) -> None:
+    wasfile = WasmFileHandler(str(Path(__file__).parent.parent / "wasm" / "add1.wasm"))
+    c = Circuit(1)
+    c.name = "test_wasm"
+    a = c.add_c_register("a", 8)
+    c.add_wasm_to_reg("add_one", wasfile, [a], [a])
+
+    b = authenticated_quum_backend_qa
+
+    c = b.get_compiled_circuit(c)
+    h = b.process_circuits(
+        [c], n_shots=10, wasm_file_handler=wasfile, language=language  # type: ignore
+    )[0]
+
+    b.get_result(h)
+
+
+@pytest.mark.skipif(skip_remote_tests_prod, reason=REASON)
+@pytest.mark.parametrize(
+    "authenticated_quum_backend_prod", [{"device_name": "H1-1SC"}], indirect=True
 )
 @pytest.mark.parametrize(
     "language",
     [
         Language.QASM,
-        # https://github.com/CQCL/pytket-quantinuum/issues/232
-        # Language.QIR,
+        Language.QIR,
     ],
 )
 @pytest.mark.timeout(120)
 def test_wasm(
-    authenticated_quum_backend: QuantinuumBackend, language: Language
+    authenticated_quum_backend_prod: QuantinuumBackend, language: Language
 ) -> None:
     wasfile = WasmFileHandler(str(Path(__file__).parent.parent / "wasm" / "add1.wasm"))
     c = Circuit(1)
     c.name = "test_wasm"
     a = c.add_c_register("a", 8)
     c.add_wasm_to_reg("add_one", wasfile, [a], [a])
 
-    b = authenticated_quum_backend
+    b = authenticated_quum_backend_prod
 
     c = b.get_compiled_circuit(c)
     h = b.process_circuits(
         [c], n_shots=10, wasm_file_handler=wasfile, language=language  # type: ignore
     )[0]
-    assert b.get_result(h)
+
+    b.get_result(h)
```

Comment:
Can we assert something useful about the result?

Reply:
See above